src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -442,7 +442,16 @@ private PoolEntry createPoolEntry()
 
          final long maxLifetime = config.getMaxLifetime();
          if (maxLifetime > 0) {
-            final long variance = maxLifetime > 60_000 ? ThreadLocalRandom.current().nextLong(10_000) : 0;
+        	 long inHours = TimeUnit.MILLISECONDS.toHours(maxLifetime);
+        	 
+        	 // Every hour will add a 10 second random factor for the maxlifetime
+        	 long randomIntervalInMillis = inHours * 10_000;
+        	 
+        	 // Support the up to 10 seconds spread out when the lifetime is less than 1 hour
+        	 if (inHours == 0) {
```

Comment:
for:
final long variance = Math.min( ThreadLocalRandom.current().nextLong( maxLifetime / 100 ), maxPoolSize * 7200);
it would be:

    age(min)    variance(sec)
    1           0.6
    5           3
    10          6
    30<---def   18
    60          36
    120         72<---max

IMO max 72 seconds separation in age would be enough for db / cput to breath easy.